src/pages/Animetoast/main.ts

```diff
@@ -0,0 +1,82 @@
+import { pageInterface } from "../pageInterface";
+import { SyncPage } from "../syncPage";
+
+export const Animetoast: pageInterface = {
+    name: "Animetoast",
+    domain: "https://www.animetoast.cc",
+    languages: ['German'],
+    type: "anime",
+    isSyncPage: function (url: string): boolean {
+        return $('.entry-title.light-title').text() != null && url.includes('?link=');
+    },
+    isOverviewPage: function (url: string): boolean {
+        return $('.entry-title.light-title').text() != null && !url.includes('?link=');
+    },
+    sync: {
+        getTitle: function (url: string): string {
+            let title = $('.entry-title.light-title').text().split(' ');
+            if(!(title.length >= 2))return title.join(' ');
+            for(let i = title.length - 1; i >= title.length - 2; i--){
+                const t = title[i];
+                t.match(/(Ger|Sub|Dub)/gi) ? title.splice(i, 1) : null;
+            }
+            return title.join(' ');
+        },
+        getIdentifier: function (url: string): string {
```

Comment:
I would recomment to use the url path as identifier
animetoast.cc/dagashi-kashi-2-ger-sub/ -> dagashi-kashi-2-ger-sub

Reply:
there are also subpages that are not animes like https://www.animetoast.cc/wochenplan-3/ is that still okay ?

Reply:
On these pages it should not even execute. You can do that over `isOverviewPage` and `isSyncPage`, if both are false then it wont do anything. Normaly we check for special elements on the page that are only on the overview page like the episode list or something like that

Reply:
ahh okay, good, did it in the fixed version how you suggested